### PR TITLE
ADD: pylint workflow

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Pylint
+
+on: [push]
```
Comment (iTitus) [inline .github/workflows/pylint.yml:3]: branch selector missing

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/pylint.yml (hunk 1 above)
Comment (Segelzwerg) [reply .github/workflows/pylint.yml:3]: Somehow I can't add your suggestion, I think it is already committed. However it doesn't show up in the commit history.

Comment (Segelzwerg) [reply .github/workflows/pylint.yml]: Never mind, my bet. Added it though

**Comment by Segelzwerg** [comment]
We need should set up a .pylintrc and disable module doc strings.

File: .github/workflows/pylint.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Pylint
+
+on:
+  push:
+    branches: [ master ]
```
Comment (iTitus) [inline .github/workflows/pylint.yml:5]:
```suggestion
    branches: [ main]
```

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/pylint.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: Pylint
+
+on:
+  push:
+    branches: [ main]
```
Comment (iTitus) [inline .github/workflows/pylint.yml:5]:
```suggestion
    branches: [main]
```

**Review by iTitus — CHANGES_REQUESTED** [review]

**Review by iTitus — APPROVED** [review]

**Pull request merged**